### ISSUE TITLE
Compiling pngs to bin format with libdragon?

### ISSUE THREAD

**Issue by neonelemental** [tristanpenman/n64-heart-rust#3]
I'm in the process of poking around with the examples directory by creating my own example to push my learning along.  I got the slide-tool example built and running in ares.  I decided to create a new example with the same structure as the slide-tool example. I want to add a new image to the example, but it looks like there's no details on how to go about taking the .png files and converting them to .bin files that can be included in the rust binary.

I'm somewhat new to Rust (novice) and completely new to n64 development.  It wasn't obvious to me if I needed to use the libdragon-cli to run `mksprite` on the png and then move that into the directory or if this is just vanilla rust png -> bin conversion.  I figured I'd open an issue and see if you could provide guidance.  Happy to open a PR to update the readme with more details on this unless it's in one of the other readmes in the repo.

**Comment by tristanpenman** [comment]
Hi @neonelemental, thanks for taking the time to raise an issue. This is definitely more confusing than it needs to be. It turns out I forgot to update the links after changing the order of the examples. Even then, it needs more coverage in the README.

I'll try to push an update this weekend, but in the meantime, you'll find a tool included in the `tools/texture-converter` directory that can be used to convert from PNG format to the RGBA5551 format used in the `slide-tool` example. This is how I generated the .bin files used for that example.

I'll let you know when the README has been updated 🙂
